samples/core/Miscellaneous/Logging/Logging/SqlCommandLoggingProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace EFLogging
+{
+    class SqlCommandLoggingProvider :
+        ILoggerProvider,
+        ILogger
+    {
+        public ILogger CreateLogger(string categoryName)
+        {
+            if (categoryName == DbLoggerCategory.Database.Command.Name)
+            {
+                return this;
+            }
+
+            return NullLogger.Instance;
+        }
+
+        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            if (eventId.Id != RelationalStrings.LogRelationalLoggerExecutedCommand.EventId.Id)
```

Comment:
So i note this is now broken in the latets EF beta. What is the replacement for `RelationalStrings.LogRelationalLoggerExecutedCommand`?

Reply:
seems it is now `RelationalEventId.CommandExecuting`